core/domain/base_model_validators.py

```diff
@@ -28,8 +28,10 @@
 from core.platform import models
 import feconf
 import python_utils
+import utils
 
-(base_models,) = models.Registry.import_models([models.NAMES.base_model])
+(base_models, user_models,) = models.Registry.import_models(
+    [models.NAMES.base_model, models.NAMES.user])
```

Comment:
nit:
```suggestion
(base_models, user_models) = models.Registry.import_models(
    [models.NAMES.base_model, models.NAMES.user])
```

Reply:
There seems to be a lot of instances where a comma is there at the end of such an import. Done.

Reply:
Yep, it is only needed in cases where there is only one model import, when there are multiple it can be removed.